proto/p4/config/v1/p4info.proto

```diff
@@ -65,6 +86,8 @@ message PkgInfo {
   string contact = 7;
   // url for more information, e.g. "http://support.p4.org/ref/p4/switch.p4_v1.0"
   string url = 8;
+  // Annotation map derived from raw annotations
+  map <string,AnnotationBody> anno_map = 9;
```

Comment:
Could we wrap the `map` in a message maybe, since it is used in quite a few places?

Reply:
Originally I had that, it looked nicer in the protobuf definition, but it added another message layer which has only one field, i.e. not a lot of value. Do you still feel it's worth it?

Reply:
OK, I added wrapper  AnnotationMap and named  the `map` field within `annotations`. The protobuf looks nice and it has symmetry with the deprecated '`annotations` field.